packages/admin/src/form/FinalFormContextProvider.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+import { FieldMetaState } from "react-final-form";
+
+export interface FinalFormContext {
+    shouldScrollToField: ({ fieldMeta }: { fieldMeta: FieldMetaState<any> }) => boolean;
+    shouldShowFieldError: ({ fieldMeta }: { fieldMeta: FieldMetaState<any> }) => boolean;
+    shouldShowFieldWarning: ({ fieldMeta }: { fieldMeta: FieldMetaState<any> }) => boolean;
+}
+
+const defaultFinalFormContext: FinalFormContext = {
+    shouldScrollToField: () => false,
+    shouldShowFieldError: ({ fieldMeta }) => !!fieldMeta?.touched,
+    shouldShowFieldWarning: ({ fieldMeta }) => !!fieldMeta?.touched,
+};
+
+const FinalFormContext = React.createContext<FinalFormContext>(defaultFinalFormContext);
```

Comment:
Is the naming of this context good? I mean, it's not *the* final-form context, ist "just" a small addition.

Reply:
this context can be described as :

- The values for this context can be set via `FinalForm`
- It's a context to be consumed in `Field` and `FieldContainer`
- the context is made specifically for final form (arguments in function are final-form-field-meta objects)
- more information that is useful for Field/FieldContainer might be added in the future to this context.

 >it's not the final-form context, ist "just" a small addition.
 
"The" final-form context would be found in the package `final-form`.
This final-form-context lives in `@comet/admin`.

My intention was to use a wide term, to keep it open to be extended. 

I am also not 100% satisfied with the name.
I don't want it to name `CometFinalFormContext`. (We do not have a CometFinalForm)
and I don't want to derive the name from the context-consumers `FinalFormFields`.

Reply:
:+1: